### DISCUSSION TITLE
Does `auto` support `pnpm` & `nx` based monorepos?

### DISCUSSION THREAD

**Discussion by allbetter-max** [intuit/auto#2281] [General]
Hi guys,

Based on this answer https://github.com/intuit/auto/discussions/1861 it seems that `auto` has built-in support for `lerna`. 

I have a monorepo with several nodejs projects that I would like to release as docker files using `auto`.

I'm using `nx affected` as the main driver for operating on changes in PR's and `pnpm` for managing the workspace and internal project dependencies (both npm and local packages).

How does `auto` fits into the mix?

More concertedly, how developer friendly the experience will be if I want to have the developer merge a `feat(app1): Adding a Cool New Thing` and `auto` will know to build add a semver+0.1.0 to my `app1` project.

**Comment by allbetter-max** [comment]
Cross linking to an open issue https://github.com/intuit/auto/issues/2149

**Comment by allbetter-max** [comment]
Cross linking to an open issue https://github.com/intuit/auto/issues/1698

**Comment by lloydjatkinson** [comment]
I would be interested in this too

**Reply by hipstersmoothie** [reply]
The last time I tried pnpm didn't require anything special

**Reply by hipstersmoothie** [reply]
Not sure about nx. If it has a package json it should be fine

**Reply by wladpaiva** [reply]
The problem is not the pnpm itself. It's the `workspace` syntax. 
So, if you are not running a monorepo, pnpm works fine

**Reply by hipstersmoothie** [reply]
ah maybe there is some update we need to do with lerna?
